dandiapi/api/services/audit/__init__.py

```diff
@@ -106,6 +106,7 @@ def create_zarr(*, dandiset: Dandiset, user: User, zarr_archive: ZarrArchive) ->
     details = {
         'zarr_id': str(zarr_archive.zarr_id),
         'name': zarr_archive.name,
+        'embargoed': dandiset.embargoed,
```

Comment:
@waxlamp does this make sense? I figured it's good to store this since I don't know if it's readily available on the zarr otherwise.

Reply:
Alright, I've removed that commit.